scripts/create-import-wrappers.mts

```diff
@@ -1,15 +1,35 @@
 import { writeFileSync } from "node:fs";
+import { basename, join } from "node:path";
+
+type LocationsRegistry =
+  | string[]
+  | {
+      [x: string]: LocationsRegistry;
+    };
```

Comment:
Use Record<> instead here

Reply:
Can't. TypeScript complains about a circular reference if I use Record, but not with this version.